README.md

```diff
@@ -117,6 +117,28 @@ this will overwrite them.
     event-name: ${{ github.event_name }}
 ```
 
+### Running Locally
+
+It can be useful to run a build outside of CI.
+So, this action can also be installed locally
+to kick off a CodeBuild project from your git sandbox.
+You could push your changes to an open PR,
+but if you only want to test one project this may be faster.
+Running this way expects you to use git to select the
+commit you want to test (by checking it out).
+
+```
+npx @aws-actions/codebuild-run-build -p ProjectName -r remoteName
+```
+
+This will take the HEAD of the current git branch
```

Comment:
This contradicts the above "check out the commit you want to test".

Looking at the code, I think that the "just uses whatever commit you have checked out" is accurate.

Reply:
I think your words are better.
But if you check out a specific commit,
you are in a "detached head",
Which is kind of like a branch where the HEAD is the commit you have checked out.

The hard case is:
I have a branch that is published and I'm both ahead and behind.
I have both staged and un-staged changes.

I could do magic to push and pop the uncommitted changes,
but that could go wrong too many ways.

What I want to handle is:
"You have pending changes that you have not yet pushed,
you want to run CodeBuild on these changes before kicking off a full CI run"

Reply:
>  But if you check out a specific commit,
you are in a "detached head",
Which is kind of like a branch where the HEAD is the commit you have checked out.

![technically correct](https://www.memesmonkey.com/images/memesmonkey/fe/fe5fc8e3c2fa7da0278c40fefe3a4831.jpeg)

I think the words as they are now are good. Let's not force the reader to have to understand the intricacies of git esoterica to know what we mean. :)